death_clock_site/death_clock/settings.py

```diff
@@ -40,7 +40,7 @@
     'death_clock_app.apps.DeathClockAppConfig'
 ]
 
-MIDDLEWARE = [
+MIDDLEWARE_CLASSES = [
```

Comment:
I believe in the updated [django version](https://docs.djangoproject.com/en/1.11/topics/http/middleware/#middleware) it is just `MIDDLEWARE`